spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -32,6 +32,13 @@
       expect(response.content_type).to eq('application/json')
       expect(response).to match_response_schema('institutions')
     end
+
+    it 'search returns case-insensitive results' do
+      get :index, name: 'CHICAGO', version: 1
```

Comment:
ED actually publishes a great school alias list that catch situations like this. We use the alias list at CFPB and is already in the raw data ingested by the data service.